src/definitions/elements/input.less

```diff
@@ -455,20 +455,34 @@
   margin: 0;
 }
 
-/* Button on Right */
+/* Input when ui Left*/
+.ui[class*="left action"].action.input > input {
```

Comment:
the additional `.action` can be removed ([class*="left action"] implies it already)
->
```css
.ui[class*="left action"].input > input {
```